apps/site/src/app/(main)/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx

```diff
@@ -29,7 +29,11 @@ export default async function ConfirmationDetails({
 			<Button
 				className="text-2xl"
 				text={isLoggedIn ? "Proceed to Application" : "Log in to Apply"}
-				href={isLoggedIn ? `${continueHREF}?prefaceAccepted=true` : "/login"}
+				href={
+					isLoggedIn
+						? `${continueHREF}?prefaceAccepted=true`
+						: `/login?application=${continueHREF.substring(1)}`
```

Comment:
Nitpick: might be good to rename `continueHREF` to a less complicated name, e.g. application URL